src/load_save.c

```diff
@@ -177,6 +178,20 @@ void LoadPlayerParty(void)
         gPlayerParty[i] = gSaveBlock1Ptr->playerParty[i];
 }
 
+void SaveChangesToPlayerParty(void)
```

Comment:
This function only gets called once, and only if `B_VAR_SKY_BATTLE` is set. Maybe it's better to just move it to `battle_setup.c` where it's called?

Reply:
fca1d479750551aa9384927a93684fc04a40ff46